zubhub_frontend/zubhub/src/assets/js/styles/components/staff_pick/staffPickStyles.js

```diff
@@ -13,6 +13,10 @@ const styles = theme => ({
   projectGridStyle: {
     marginBottom: '2em',
   },
+  MessagePrimaryStyle: {
+    fontSize: '2.5rem',
+    fontWeight: 1000,
```

Comment:
it shouldn't be 1000 🤔

Reply:
Since the aim was to make the heading for the Featured Project section the same as the headings on other sections on the landing page, I have used the same style used in those headings.

Reply:
Sorry my mistake was thinking of some other property which should be <10